src/sql/migrations.rs

```diff
@@ -497,6 +497,19 @@ item TEXT DEFAULT '');"#,
         )
         .await?;
     }
+    if dbversion < 81 {
+        info!(context, "[migration] v81");
+        sql.execute_migration(
+            r#"CREATE TABLE msgs_status_updates (
+id INTEGER PRIMARY KEY AUTOINCREMENT,
+msg_id INTEGER,
```

Comment:
when this is referring to the message that instantiated the w30app, then the
sender contact id is missing, isn't it?

Reply:
see comment below.

Reply:
meanwhile, the `payload` column is renamed ot `update_item` that contains the payload - and maybe other things in the future.

if it turns out, that columns are needed in the future, we can add them as needed, however, i assume most things can directly be added to the update_item which also has the effect that they are more self-contained and not dependent to database changes. things are changed email addresses (aeap) may or may not work for a concrete app, i think it is app-dependent what actually should happen. but i think, for an mvp, it is fine to say that this is just not supported and will result in undefined behavior.